pyvcpkg.py

```diff
@@ -93,11 +95,11 @@ def update_vcpkg(self) -> None:
             [str(self.vcpkg_path / 'bootstrap-vcpkg.bat')],
         ]
         for cmd in cmds:
-            subprocess.run(cmd, cwd=self.vcpkg_path, check=True)
+            subprocess.run(cmd, cwd=str(self.vcpkg_path), check=True)
```

Comment:
[The subprocess documentation](https://docs.python.org/3/library/subprocess.html) states that python versions >= 3.6 accept a path-like object as current working directory.
I think an `assert sys.version_info >= (3, 6), "This script does not support this version of Python"` at the top of the file would look better.
Since this file should only be invoked by builders who (should) have an up-to-date system.